documentation/release_notes.rst

```diff
@@ -36,6 +36,10 @@ New in This Release
   ``libstdc++`` version 8 or ``libc++``, ``oneapi::dpl::execution::par_unseq`` offloads
   standard parallel algorithms to the SYCL device similarly to ``std::execution::par_unseq``
   in accordance with the ``-fsycl-pstl-offload`` option value.
+- When using the dpl modulefile to initialize the user's environment and compiling with ``-fsycl-pstl-offload``
```

Comment:
Two minor updates (adding a comma and corrected the compiler name):

When using the dpl modulefile to initialize the user's environment and compiling with ``-fsycl-pstl-offload`` option of Intel® oneAPI DPC++/C++ Compiler, a linking issue or program crash may be encountered due to the directory containing libpstloffload.so not being included in the search path. Use the env/vars.sh to configure the working environment to avoid the issue.